jekyll/_cci2/testing-orbs.md

```diff
@@ -1,187 +1,132 @@
+For more information see the [Writing Inline Orbs]({{site.baseurl}}/2.0/reusing-config/#writing-inline-orbs) guide.
 
-The following steps illustrate doing basic expansion testing from the CLI:
+## Schema Validation
 
-1) Assume a simple orb in `src/orb.yml`
+To test whether an orb is valid YAML and is well-formed according to the CircleCI schema, use the CircleCI CLI command `circleci orb validate`.
 
-{% raw %}
-```yaml
-version: 2.1
+For example,given an orb with source at `./src/orb.yml`, run `circleci orb validate ./src/orb.yml` to receive feedback on whether the orb is valid and will pass through config processing. If there is an error, you will receive the first schema validation error encountered. Alternatively, you can pass STDIN rather than a file path.
 
-executors:
-  default:
-    parameters:
-      tag:
-        type: string
-        default: "curl-browsers"
-      docker:
-        - image:  circleci/buildpack-deps:parameters.tag
-
-jobs:
-  hello-build:
-    executor: default
-    steps:
-      - run: echo "Hello, build!"
-```
-{% endraw %}
+## Integration Testing
 
-2) Validate the orb with `circleci orb validate src/orb.yml`.
+Using [development orbs]({{site.baseurl}}/2.0/orb-concepts/#development-orbs), it is possible to publish changes to a live development version of an orb and use that development version within a live configuration for testing. Development versions of orbs can later be promoted to production orbs.
 
-3) Publish a dev version with `circleci orb publish src/orb.yml namespace/orb@dev:0.0.1`.
+Integrations tests validate the usage of your orb and its parameters in a live environment on CircleCI using sample project code (if required). Integration test jobs should be declared that both use the commands defined within your orb, and validate the correct functioning of those commands. See the following examples for testing a command, and testing a job.
 
-4) Include that orb in `.circleci/config.yml`:
+### Test a command
 
-{% raw %}
 ```yaml
-version: 2.1
-
 orbs:
-  hello: namespace/orb@dev:0.0.1
-
-workflows:
-  hello-workflow:
+  # import development version of orb
+  my-orb: my-namespace/my-orb@dev:my-dev-version
     jobs:
-      - hello/hello-build
-```
-{% endraw %}
-
-After running `circleci config process .circleci/config.yml` the expected result would be:
-
-{% raw %}
-```yaml
-version: 2.1
-
-jobs:
-  hello/hello-build:
-    docker:
-      - image: circleci/buildpack-deps:curl-browsers
-    steps:
-      - run:
-          command: echo "Hello, build!"
+      # create jobs for each test
+      test-my_command:
+        docker:
+          - image: cimg/base:stable
+        steps:
+          - my-orb/my_command:  # Invoking the orb command from the development orb
+              my_parameter: "value"
+          - run:
+              name: Validate my_command
+              command: |
+                if [ "$ITWORKED" != "yes" ]; then
+                echo "error" && exit 1
+                fi
```

Comment:
If I understand right this should all be the job declaration, not within the orb itself and so aligned a few tabs to the left (including the `jobs` key above)?

Reply:
Lets zoom on this, im not following.